.evergreen.yml

```diff
@@ -1,19 +1,64 @@
+stepback: true
+command_type: test
+ignore:
+  - "*.md"
+  - ".github/*"
+
+#######################################
+#              Functions              #
+#######################################
 functions:
-  fake_function:
-    - command: shell.exec
-      params:
-        script: |
-          echo "I am a fake function"
+  get-project:
+    command: git.get_project
+    type: setup
+    params:
+      directory: parsley
+
+  yarn-build:
+    command: shell.exec
+    params:
+      working_dir: parsley
+      script: |
+        export PATH=/opt/nodejs/node-v16.13.0-linux-x64/bin:$PATH
```

Comment:
Could we add this line either to `get-project` or as its own function in order to avoid repetition? Will be easier to upgrade our version of Node in the future :)

Reply:
It was not very pleased when I tried to move these scripts out (my [patch](https://spruce.mongodb.com/version/62f414eee3c3311893d459d8/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC)), so I went for an expansion instead 🤔

Reply:
So actually it looks like the problem with your patch is that `script` should be a field within the `params` field (i.e. indented under it). YAML syntax is so hard 😭. I ran a successful patch with that approach [here](https://spruce.mongodb.com/version/62f664810305b95854d1f747/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC). Mind going with that instead of the variable? I feel like it's a little simpler.

Reply:
I tried again but I couldn't get it to work 🥲 ([my patch](https://spruce.mongodb.com/version/62f667039ccd4e2d11fa5657/changes?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC))

I also tried it as a separate command in [this patch](https://spruce.mongodb.com/version/62f422d1e3c3311893d47b61/changes?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC)

I think the patch you linked may have passed because each of the commands still have individual export statements like this `export PATH=${node_path}`, or I am just very bad at copy pasting